docs/Readme.md

```diff
@@ -14,29 +14,36 @@ If you would like to see more examples of field data plug-ins, please visit our
 
 ## Change Log
 
+### AQTS 2020.3 - Framework version 2.10
+- Added support for `GageZeroFlowActivity` activities to record the gage height at zero flow.
```

Comment:
There wasn't any documentation updates for the new `GageZeroFlowActivity`, so I added it

Reply:
Oops, thanks for adding this.